src/core/api/commissioner_api.cpp

```diff
@@ -185,11 +185,11 @@ otError otCommissionerGeneratePskc(otInstance *           aInstance,
                                    const char *           aPassPhrase,
                                    const char *           aNetworkName,
                                    const otExtendedPanId *aExtPanId,
-                                   uint8_t *              aPskc)
+                                   otPskc *               aPskc)
 {
     OT_UNUSED_VARIABLE(aInstance);
```

Comment:
Should we remove `aInstance` from the argument list?

Reply:
Good idea! Since we are changing the API anyway, good time to remove `aInstance` as well!

Reply:
Changed it in new pushed commits.